src/chrome/content/rules/Zero_Day_Initiative.com.xml

```diff
@@ -1,25 +1,12 @@
 <!--
-	For other Hewlett-Packard coverage, see Hewlett-Packard.xml.
-
-
-	Some pages redirect to http.
-
+	For other Trend Micro coverage, see Trend_Micro.xml. 
```

Comment:
Please also update the references in `Trend_Micro.xml`.

Reply:
@J0WI Done. Thanks!